include/jemalloc/internal/prng.h

```diff
@@ -64,61 +64,34 @@ prng_state_next_zu(size_t state) {
  */
 
 JEMALLOC_ALWAYS_INLINE uint32_t
-prng_lg_range_u32(atomic_u32_t *state, unsigned lg_range, bool atomic) {
-	uint32_t ret, state0, state1;
-
+prng_lg_range_u32(uint32_t *state, unsigned lg_range) {
```

Comment:
So we can also delete the `#include "jemalloc/internal/atomic.h"` at the beginning of this header file?